site/en/docs/extensions/mv3/manifest/index.md

```diff
@@ -27,7 +27,7 @@ The following code shows the supported manifest keys. For additional usage infor
   <span class="token property">"<a href="/docs/extensions/mv3/manifest/description">description</a>"</span><span class="token operator">:</span> <span class="token string">"A plain text description"</span><span class="token punctuation">,</span>
   <span class="token property">"<a href="/docs/extensions/mv3/manifest/icons">icons</a>"</span><span class="token operator">:</span> <span class="token punctuation">{</span>...<span class="token punctuation">}</span><span class="token punctuation">,</span>
 
-  <span class="token comment">// Optional</span>
+  <span class="token comment">// Required for Chrome Web Store. Optional for unpacked extensions.</span>
   <span class="token property">"<a href="/docs/extensions/mv3/manifest/author">author</a>"</span><span class="token operator">:</span> <span class="token string">"developer@example.com"</span><span class="token punctuation">,</span>  
   
```

Comment:
We should explicitly say that these keys are optional

Reply:
Maybe I did this in the wrong place. Isn't there only an `"email"` key below this? It's `"email"` that makes it required.

I'm also trying to find out if it has to match the contact email on the dashboard.

Reply:
Should this information go under the ["author" key](https://pr-7146-static-dot-dcc-staging.uc.r.appspot.com/docs/extensions/mv3/manifest/author/) page?